docs-src/concepts/what-is-a-schedule.md

```diff
@@ -183,7 +183,7 @@ A Workflow started by a Schedule can obtain the details of the failure of the mo
 
 ### Limitations
 
-:::info Experimental
+:::info
 
 The Scheduled Workflows feature is available in Temporal Server version 1.18.
```

Comment:
The version info is also captured by the ssdi meta data at the top of this file.

Reply:
Can we have it say "Public Preview" - since that is the official status

Reply:
> We need to add:

I don't think we should get in the habit of documenting SDK versions of when we made features available. That they are available on the latest SDK version should be enough info IMO.

Reply:
@mbernier do we have a definition of "Public Preview" we should link to?

Reply:
@cretz I can imagine if it's not there, I might read the docs and try to use a feature, and not understand why it's not working. For that case, it would be helpful to see beforehand that I need to upgrade, and to which version.

Reply:
> I can imagine if it's not there, I might read the docs and try to use a feature, and not understand why it's not working.

How have you survived with any other feature Temporal has ever released ever until now then? If we want a new policy of documenting when features become available in SDKs and their versions in this non-changelog, we need to apply it generally and not here as the first application of this new policy.

> For that case, it would be helpful to see beforehand that I need to upgrade, and to which version.

You always need to upgrade. You always have needed to upgrade for new things. Schedules is not new here and we should have a general discussion about documentation of SDK versions in these docs if we're going to make a new policy here.

Reply:
👍 will start general discussion in slack